plantseg/__init__.py

```diff
@@ -9,3 +11,9 @@
 PLANTSEG_MODELS_DIR = ".plantseg_models"
 configs_path = os.path.join(home_path, PLANTSEG_MODELS_DIR, "configs")
 os.makedirs(configs_path, exist_ok=True)
+
+# create custom zoo if does not exist
+custom_zoo = os.path.join(os.path.expanduser("~"), ".plantseg_models", 'custom_zoo.yaml')
```

Comment:
you may want to use `PLANTSEG_MODELS_DIR` constant instead of hardcoded string `".plantseg_models"`. not a big deal though

Reply:
It is a general problem of the gui.. I removed as much as possible the hard-coded strings now most of the common locations are defined in the main init